modules/contracts/adapters/cw-staking/STATUS.md

```diff
@@ -5,7 +5,7 @@ This document describes the status of the staking adapter's integrations with di
 | Protocol | Implementation | Execution Tests | Query Tests | Notes |
 | --- | --- | --- | --- | --- |
 | Osmosis | ✅ | ✅ | ❌ | Query failing because of missing whitelist. |
-| Astroport | ✅ | ✅ | ❌ | |
+| Astroport | ✅ | ✅ | ✅ | |
 | Wynd | ✅ | ✅ | ❌ | |
-| Bow | ✅ | ✅ | ❌ | Execution not possible because it uses custom module. |
+| Bow | ✅ | ✅ | ✅ | Creating pool not possible because it uses custom module. |
```

Comment:
So execution is not tested? The note says something is not possible, what is the consequence of that?

Reply:
> So execution is not tested? The note says something is not possible, what is the consequence of that?

Execution is tested, but on existing pool, instead of freshly-made

Reply:
Consequence of that is that we rely on "test pool" to exist relatively unchanged